pkg/api/common/versions.go

```diff
@@ -178,11 +178,12 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.17.2":         false,
 	"1.17.3":         true,
 	"1.17.4":         true,
-	"1.18.0-alpha.1": true,
-	"1.18.0-alpha.2": true,
-	"1.18.0-alpha.3": true,
-	"1.18.0-alpha.5": true,
-	"1.18.0-beta.1":  true,
+	"1.18.0-alpha.1": false,
+	"1.18.0-alpha.2": false,
+	"1.18.0-alpha.3": false,
+	"1.18.0-alpha.5": false,
+	"1.18.0-beta.1":  false,
```

Comment:
Should we add `false` entries for 1.18.0-beta.2 and 1.18.0-rc.1? AKS Engine never supported either one due to bugs, but we could add them in order to reference the issues.

Reply:
Let's do, good call.